README.md

```diff
@@ -76,19 +76,19 @@ end
 
 The following resources are available in the InSpec Azure Resource Pack
 
-- [azure_monitor_activity_log_alert](docs/resources/azure_monitor_activity_log_alert.md.erb)
 - [azure_monitor_activity_log_alerts](docs/resources/azure_monitor_activity_log_alerts.md.erb)
-- [azure_monitor_log_profile](docs/resources/azure_monitor_log_profile.md.erb)
-- [azure_monitor_log_profiles](docs/resources/azure_monitor_log_profiles.md.erb)
+- [azurerm_network_security_groups](docs/resources/azurerm_network_security_groups.md.erb)
 - [azure_resource_groups](docs/resources/azure_resource_groups.md.erb)
-- [azure_security_center_policies](docs/resources/azure_security_center_policies.md.erb)
-- [azure_security_center_policy](docs/resources/azure_security_center_policy.md.erb)
+- [azurerm_security_center_policies](docs/resources/azurerm_security_center_policies.md.erb)
+- [azure_monitor_log_profile](docs/resources/azure_monitor_log_profile.md.erb)
 - [azurerm_network_security_group](docs/resources/azurerm_network_security_group.md.erb)
-- [azurerm_network_security_groups](docs/resources/azurerm_network_security_groups.md.erb)
-- [azurerm_network_watcher](docs/resources/azurerm_network_watcher.md.erb)
-- [azurerm_network_watchers](docs/resources/azurerm_network_watchers.md.erb)
+- [azure_monitor_log_profiles](docs/resources/azure_monitor_log_profiles.md.erb)
+- [azurerm_security_center_policy](docs/resources/azurerm_security_center_policy.md.erb)
 - [azurerm_virtual_machine](docs/resources/azurerm_virtual_machine.md.erb)
 - [azurerm_virtual_machine_disk](docs/resources/azurerm_virtual_machine_disk.md.erb)
+- [azurerm_network_watcher](docs/resources/azurerm_network_watcher.md.erb)
+- [azure_monitor_activity_log_alert](docs/resources/azure_monitor_activity_log_alert.md.erb)
+- [azurerm_network_watchers](docs/resources/azurerm_network_watchers.md.erb)
 - [azurerm_virtual_machines](docs/resources/azurerm_virtual_machines.md.erb)
```

Comment:
Should we alphabetize this list going forward?

Reply:
Let's get #89 merged first. After the ordering gets cleaned up here I think we are good to merge.